lib/meadow/data/schemas/work.ex

```diff
@@ -77,6 +77,15 @@ defmodule Meadow.Data.Schemas.Work do
             nil -> %{}
             collection -> %{id: collection.id, title: collection.name}
           end,
+        file_sets:
+          work.file_sets
+          |> Enum.map(fn file_set ->
+            %{
+              id: file_set.id,
+              accession_number: file_set.accession_number,
+              label: file_set.metadata.description
```

Comment:
The label is a actually the label. But I think my Manifest PR adds it in....
The label is populated with original_filename on ingest and then is editable.

Should we just wait to merge mine in and then you can rebase?